app/src/main/java/com/duckduckgo/app/cta/ui/CtaViewModel.kt

```diff
@@ -283,6 +283,11 @@ class CtaViewModel @Inject constructor(
         }
     }
 
+    private fun isSiteNotAllowedForOnboarding(url: String?): Boolean {
+        val isPrivacyProSite = url == "https://duckduckgo.com/pro"
```

Comment:
This can still return false when it shouldn't, the URL can have arguments or more paths and then this won't be true. It should really be doing something similar to `shouldLaunchPrivacyProForUrl`, first check the eTLD is duckduckgo.com and then check the first path is pro.

Reply:
Thanks for the feedback @marcosholgado ! I'll commit the changes

Reply:
I don't think we should use that same method exactly, we probably want to skip onboarding even though a subscription is not active

Reply:
Done. Thank you both :)